### PR TITLE
Manage sshd from admin ui

### PR DESCRIPTION
**Pull request by gabriel-v**
This patch makes the `Admin UI > Network > SSH` tab work as expected.

I will still have to change `ci/prepare-image-for-testing` to open up the image for first boot tests and testing login.

### CODE REVIEW

File: ansible/roles/common/tasks/sshd.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  template:
+    src: sshd_config
+    dest: /etc/ssh/sshd_config
+  tags:
+    - configure
+
+- name: Enable ssh server if configured
+  service:
+    name: ssh
+    enabled: "{{ liquid_ssh.enabled }}"
```
Comment (mgax) [inline ansible/roles/common/tasks/sshd.yml]: Elegant.

File: ansible/roles/common/templates/sshd_config [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Package generated configuration file
```
Comment (mgax) [inline ansible/roles/common/templates/sshd_config]: Let's lose all these comments.

File: ansible/roles/devel/tasks/main.yml [hunk 3]
```diff
@@ -13,14 +13,9 @@
         - vim
         - htop
         - zsh
+        - sl
```
Comment (mgax) [inline ansible/roles/devel/tasks/main.yml]: :))

**Comment by gabriel-v** [comment]
See relevant builds at https://jenkins.liquiddemo.org/blue/organizations/jenkins/liquidinvestigations%2Fsetup/activity?branch=PR-109

File: ansible/roles/common/templates/sshd_config [hunk 4]
```diff
@@ -0,0 +1,77 @@
+Port {{ liquid_ssh.port }}
+#ListenAddress ::
```
Comment (mgax) [inline ansible/roles/common/templates/sshd_config]: Really, let's lose all the comments, they are just noise. We can always look up sshd configuration options in the man page.

**Pull request merged**